runtime/interface.cpp

```diff
@@ -601,6 +607,20 @@ void f$fastcgi_finish_request(int64_t exit_code) {
       break;
     }
     case QUERY_TYPE_HTTP: {
+      // Run custom headers handler before body processing
+      if (headers_custom_handler_function && !headers_sent) {
+        headers_sent = true;
+        headers_custom_handler_function();
+        // In `headers_custom_handler_function` might be happened `f$flush` invocation
+        // First of all, we have to accumulate response data from buffers
+        ob_total_buffer = ob_merge_buffers();
```

Comment:
here we call `ob_merge_buffers()` twice, even on usual execution path, is it idempotent?

Reply:
Invocation moved into more appropriate place, fixed